assemblyline/filestore/__init__.py

```diff
@@ -2,12 +2,11 @@
 
 import json
 import logging
-from typing import AnyStr, TYPE_CHECKING, Optional, Tuple
-from urllib.parse import urlparse, parse_qs, unquote
+from typing import TYPE_CHECKING, AnyStr, Optional, Tuple
+from urllib.parse import parse_qs, unquote, urlparse
 
 import elasticapm
-
-from assemblyline.common.exceptions import get_stacktrace_info
+from assemblyline.common.exceptions import FileStoreException, get_stacktrace_info
```

Comment:
This line breaks the builds, can you remove it please

Reply:
https://dev.azure.com/CybercentreCanada/Assemblyline/_build/results?buildId=59389&view=logs&j=2750ccff-bc84-550d-fba1-e59c7681252a&t=91cf34b1-edd2-50d9-08e9-ab00b0ff4455&l=188